app/javascript/src/components/Hero.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { Button } from "neetoui";
+
+import { FEATURE_LIST } from "./constants";
+
+const Features = ({ features }) => {
+  return (
+    <ul className="list-disc">
+      {features.map((feature, index) => (
+        <li key={index} className="border-b border-gray-100 py-3 last:border-0">
+          {feature}
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+const Hero = () => (
+  <div className="flex h-screen flex-row items-center justify-center">
+    <div className="m-auto flex max-w-3xl flex-col items-center justify-center p-8">
+      <h1 className="mb-3 text-4xl font-bold">Wheel</h1>
+      <p className="mx-auto mb-6 text-center text-lg text-gray-800">
+        Don&apos;t reinvent the wheel. Use sane defaults to bootstrap your
+        react-rails project!
+      </p>
+      <Features features={FEATURE_LIST} />
+      <div className="mt-6 flex items-center justify-center">
+        <Button size="large" type="primary" to="/login" label="Login" />
+      </div>
+    </div>
+  </div>
+);
+
+export default Hero;
```

Comment:
You can memoize this component.

Reply:
@yedhink We can memoize this component but we should be careful here as all code in wheel sends a message about the code practices we use at BigBinary. 

This component will only be rendered once before login. It won't be rendered after the login. As this component is not bulky and not being rendered repeatedly there won't be any significant performance gains.

Please let me know your thoughts on this.

Reply:
That's a fair point. I guess we don't need to memoize then because it's rendered only before login.